yolo_angles_error_adjustment: 0.0 #radians
marker_base_diameter_for_filtering: 0.4572 #meters 
marker_diameter_for fitlering: 0 #unknown
max_lidar_range: 29 #meters - should be slightly less than real range, or accurate range
min_lidar_range: 0.1 #meters - should be slight larger than real, or accurate range
prop_coord_range: 0.1 #meters, used to create a 'bubble' around identified prop so that it doesn't remap the same prop
realsense_fov: 1.204277184 #radians
realsense_res_x: 1920 #pixel width of image frames